pkg/webui/console/containers/device-onboarding-form/device-provisioning-form-section/index.js

```diff
@@ -38,36 +40,70 @@ import DeviceRegistrationFormSection, {
   initialValues as registrationInitialValues,
 } from './device-registration-form-section'
 
+const msg = defineMessages({
+  continue:
+    'To continue, please enter the JoinEUI of the end device so we can determine onboarding options',
+  confirm: 'Confirm',
+  reset: 'Reset',
+  registration: '✔ This end device can be registered on the network',
+  claiming: '✔ This end device can be claimed from its current owner',
+})
+
 const initialValues = merge(
   {
     ids: {
       join_eui: '',
     },
+    _claim: '',
```

Comment:
```suggestion
    _claim: undefined,
```
An empty string is not a good default value for this. You can use `undefined` here since the field does not map to any actual field and it is more fitting for this scenario.